site/features.md

```diff
@@ -53,6 +54,29 @@ now re-use the same tools and techniques when deploying applications
 as containers as we would have done when deploying them 'on metal' in
 our data centre.
 
+### <a name="ipam"></a>IP Address Management
+
+Weave can assign unique IP addresses to each container across the
+network.  To make this work, weave must be told what range of
+addresses to allocate from, on each host.  For example:
```

Comment:
The wording here is ambiguous. "on each host" could be be interpreted as meaning that each host must be given a *separate* range to allocate from. Whereas usually the specified range should be the same across hosts (what happens if it isn't?).